src/main/native/blake3_jni.cc

```diff
@@ -0,0 +1,94 @@
+// Copyright 2023 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <jni.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "external/blake3/c/blake3.h"
```

Comment:
@meteorcloudy Can you show me how to correctly refer to this header when it's coming from bzlmod?

The bzlmod tests are failing with error:
```
src/main/native/blake3_jni.cc:19:10: fatal error: external/blake3/c/blake3.h: No such file or directory
   19 | #include "external/blake3/c/blake3.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

because bzlmod is putting this source in a directory like `external/blake3~1.3.3/...` which causes them not to be found.

Thanks!

Reply:
All repository roots are also include directories, so this works:

```suggestion
#include "c/blake3.h"
```

Reply:
Ah, amazing, thank you @fmeum!